python/revng/cli/analyze.py

```diff
@@ -0,0 +1,25 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+from .commands_registry import Command, Options, commands_registry
+from .support import build_command_with_loads, collect_files, interleave, run
+
+
+class AnalyzeCommand(Command):
+    def __init__(self):
+        super().__init__(("analyze",), "revng analyzer", False)
+
+    def register_arguments(self, parser):
+        pass
+
+    def run(self, options: Options):
+        pipelines = collect_files(options.search_prefixes, ["share", "revng", "pipelines"], "*.yml")
+        pipelines_args = interleave(pipelines, "-P")
+        command = build_command_with_loads(
+            "revng-analyze", pipelines_args + options.remaining_args, options
+        )
+        return run(command, options)
```

Comment:
I guess this might be a good time to factor this class  out.

Reply:
since this affects the original pipeline tool tool i will add this change as a extra commit instead of modifying the various commits